plaid/requester.py

```diff
@@ -20,12 +20,25 @@ def _requests_http_request(url, method, data, timeout=DEFAULT_TIMEOUT):
             headers={
                 'User-Agent': 'Plaid Python v{}'.format(__version__),
             },
-            timeout=timeout,
-        )
+            timeout=timeout, )
     else:
-        raise Exception(
-            'Invalid request method {}'.format(method)
-        )
+        raise Exception('Invalid request method {}'.format(method))
+
+
+async def _requests_http_request_async(url,
```

Comment:
To maintain `python2.x` support, I believe we'll need to place `_requests_http_request_async` and `http_request_async` into a separate file